.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,97 @@
+name: CI
+
+# target branches for the ci workflow
+on:
+  push:
+    branches: ['main', 'develop']
+  pull_request:
+    branches: ['main', 'develop']
+
+jobs:
+  install:
+    name: Install
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the source code
+      - name: Checkout source code
+        uses: actions/checkout@v3
+      # Cache npm dependencies
+      - name: Cache node dependencies
+        uses: actions/cache@v3
+        id: node-cache
+        with:
+          path: |
+            node_modules
```

```suggestion
          path: node_modules
```

Pipe syntax is used when there are multiple paths to consider for the cache key. here there is only node_modules so the inline syntax is better. You'll need to also change the syntax in the steps loading the cache below